docs/data/sql_functions_zh.yml

```diff
@@ -0,0 +1,820 @@
+      时区的格式应该是缩写如 “PST”，全名如 “America/Los_Angeles”，或自定义ID如 “GMT-08:00”。例如
+      `CONVERT_TZ('1970-01-01 00:00:00', 'UTC', 'America/Los_Angeles')` 返回 `'1969-12-31 16:00:00`'。
+  - sql: FROM_UNIXTIME(numeric[, string])
+    description: |
+      以字符串格式返回数字参数的表示形式（默认为 'yyyy-MM-dd HH:mm:ss'）。numeric 是一个内部时间戳值，表示自
+      '1970-01-01 00:00:00' UTC 以来的秒数，由 UNIX_TIMESTAMP() 函数生成。返回值以会话时区表示（在 TableConfig
+      中指定）。例如，如果在 UTC 时区，FROM_UNIXTIME(44) 返回 '1970-01-01 00:00:44'，如果在 'Asia/Tokyo' 时区，
+      则返回 '1970-01-01 09:00:44'。
+  - sql: UNIX_TIMESTAMP()
+    description: 以秒为单位获取当前的 Unix 时间戳。此函数不是确定性的，这意味着将为每个记录重新计算该值。
+  - sql: UNIX_TIMESTAMP(string1[, string2])
+    description: |
+      使用表配置中指定的时区将格式为 string2 的日期时间字符串 string1（如果未指定默认情况下：yyyy-MM-dd HH:mm:ss）
+      转换为 Unix 时间戳（以秒为单位）。
+  - sql: TO_DATE(string1[, string2])
+    description: 将格式为 string2（默认为 'yyyy-MM-dd'）的字符串 string1 转换为日期。
+  - sql: TO_TIMESTAMP_LTZ(numeric, precision)
+    table: toTimestampLtz(numeric, PRECISION)
+    description: |
+      将纪元秒或纪元毫秒转换为 TIMESTAMP_LTZ，有效精度为 0 或 3，0 代表 `TO_TIMESTAMP_LTZ(epochSeconds, 0)`，
+      3 代表` TO_TIMESTAMP_LTZ(epochMilliseconds, 3)`。
+  - sql: TO_TIMESTAMP(string1[, string2])
+    description: 将 'UTC+0' 时区下格式为 string2（默认为：'yyyy-MM-dd HH:mm:ss'）的字符串 string1 转换为时间戳。
+  - sql: CURRENT_WATERMARK(rowtime)
+    description: |
+      返回给定时间列属性的当前水印，或者 NULL。如果管道中的当前操作没有可用的上游操作的公共水印。函数的返回类型被推断为与提
+      供的时间列属性匹配，但调整后的精度为 3。例如如时间列属性为 TIMESTAMP_LTZ(9)，则函数将返回 TIMESTAMP_LTZ(3)。
+
+      请注意，此函数可以返回NULL，您可能必须考虑这种情况。例如，如果您想过滤掉后期数据，您可以使用：
+      ```
+      WHERE
+        CURRENT_WATERMARK(ts) IS NULL
+        OR ts > CURRENT_WATERMARK(ts)
+      ```
+
+conditional:
+  - sql: |
+      CASE value
+        WHEN value1_1 [, value1_2]* THEN RESULT1
+        (WHEN value2_1 [, value2_2 ]* THEN result_2)*
+        (ELSE result_z)
+      END
+    description: |
+      当第一个时间值包含在 (valueX_1, valueX_2, ...) 中时，返回 resultX。当没有值匹配时，如果提供则返回 result_z，
+      否则返回 NULL。
+  - sql: |
+      CASE
+        WHEN condition1 THEN result1
+        (WHEN condition2 THEN result2)*
+        (ELSE result_z)
+      END
+    description: 满足第一个条件 X 时返回 resultX。当不满足任何条件时，如果提供则返回 result_z，否则返回 NULL。
+  - sql: NULLIF(value1, value2)
+    description: |
+      如果 value1 等于 value2 返回 NULL；否则返回 value1。例如 `NULLIF(5, 5)` 返回 NULL；`NULLIF(5, 0)` 返回 5。
+  - sql: COALESCE(value1, value2 [, value3]*)
+    description: 从 value1, value2, ... 返回第一个不为 NULL 的值。例如 `COALESCE(3, 5, 3)` 返回 3。
+  - sql: IF(condition, true_value, false_value)
+    description: 如果满足条件，则返回 true_value，否则返回 false_value。例如 `IF(5 > 3, 5, 3)` 返回 5。
+  - sql: IFNULL(input, null_replacement)
+    table: input.ifNull(nullReplacement)
+    description: |
+      如果输入为 NULL，则返回 null_replacement；否则返回输入。与 COALESCE 或 CASE WHEN 相比，此函数返回的数据类型
+      在可空性方面非常具体。返回的类型是两个参数的公共类型，但只有在 null_replacement 可为空时才能为空。该函数允许将可
+      为空的列传递到使用 NOT NULL 约束声明的函数或表中。例如 `IFNULL(nullable_column, 5)` 一定不返回 NULL。
```

Comment:
```suggestion
      为空的列传递到使用 NOT NULL 约束声明的函数或表中。例如 `IFNULL(nullable_column, 5)` 一定不返回 NULL。
```

Reply:
It seems no different？

Reply:
nothing different.  Keep your raw thranslation , please.